oaknational/Oak-Web-Application
# LESQ-893: Conditional rendering of tier-title

## Description

Music year: 1967

- Conditionally renders tier based on whether page is canonical or not

## Issue(s)

Fixes #LESQ-893

https://www.notion.so/oaknationalacademy/Canonical-lesson-displays-a-tier-even-though-it-should-be-completely-agnostic-3a6b4706da0f44bab4eba1951d0d3515

## How to test

1. Go to https://deploy-preview-2932--oak-web-application.netlify.thenational.academy/teachers/lessons/atoms-elements-and-compounds#slide-deck
2. You should see no tier title in heading metadata

## Screenshots

How it used to look (delete if n/a):
<img width="1485" alt="Screenshot 2024-10-25 at 16 25 19" src="https://github.com/user-attachments/assets/eb1073e7-7073-434b-a7fe-5f1bc7371454">

How it should now look:
<img width="1332" alt="Screenshot 2024-10-25 at 16 24 17" src="https://github.com/user-attachments/assets/014f769c-941e-43da-878f-0ab51dce7d8d">

## Checklist
